htdocs/core/menus/standard/eldy.lib.php

```diff
@@ -85,9 +85,9 @@ function print_eldy_menu($db, $atarget, $type_user, &$tabMenu, &$menu, $noout =
 		'position' => 10,
 		'id' => $id,
 		'idsel' => 'home',
-		'classname' =>  $classname = ($_SESSION["mainmenu"] && $_SESSION["mainmenu"] == "home") ? 'class="tmenusel"' : 'class="tmenu"',
+		'classname' =>  $classname = (!isset($_SESSION["mainmenu"]) && $_SESSION["mainmenu"] == "home") ? 'class="tmenusel"' : 'class="tmenu"',
```

Comment:
I think test here should be 
!empty($_SESSION["mainmenu"])
instead of 
!isset

Reply:
Done

Reply:
Same on line 88

Reply:
Oops, my bad, I changed all occurences